themes/grass/layouts/partials/footer.html

```diff
@@ -1,3 +1,88 @@
+<footer class="section grass-blue-bg text-light pt-5 pb-3">
+  <div class="container">
+    <div class="row">
+
+      <!-- Logo -->
+      <div class="col-md-2 text-center text-md-left mb-4 mb-md-0">
+        <img src="{{.Site.BaseURL}}/images/logos/gs-white-small.png" alt="GRASS GIS logo" style="max-width: 100px;">
+      </div>
+
+      <!-- Quick Links -->
+      <div class="col-md-2 text-center text-md-left mb-4 mb-md-0">
+        <h6 class="text-uppercase font-weight-bold">About Us</h6>
+        <ul class="list-unstyled small">
+          <li><a href="/about/team" class="text-light">Team</a></li>
+          <li><a href="/about/roadmap" class="text-light">Roadmap</a></li>
+          <li><a href="/about/governance" class="text-light">Governance</a></li>
+        </ul>
+      </div>
+
+      <div class="col-md-2 text-center text-md-left mb-4 mb-md-0">
+        <h6 class="text-uppercase font-weight-bold">Learn</h6>
+        <ul class="list-unstyled small">
+          <li><a href="/learn/manuals" class="text-light">Documentation</a></li>
+          <li><a href="https://grass-tutorials.osgeo.org/" class="text-light">Tutorials</a></li>
+          <li><a href="/learn/books" class="text-light">Books</a></li>
+        </ul>
+      </div>
+
+      <div class="col-md-2 text-center text-md-left mb-4 mb-md-0">
+        <h6 class="text-uppercase font-weight-bold">Support</h6>
+        <ul class="list-unstyled small">
+          <li><a href="/support/community" class="text-light">Community</a></li>
+          <li><a href="/support/commercial" class="text-light">Commercial</a></li>
+          <li><a href="/support/commercial" class="text-light">Contact Us</a></li>
+        </ul>
+      </div>
+
+      <div class="col-md-2 text-center text-md-left mb-4 mb-md-0">
+        <h6 class="text-uppercase font-weight-bold">Contribute</h6>
+        <ul class="list-unstyled small">
+          <li><a href="/support/development" class="text-light">Development</a></li>
```

Comment:
Link isn't valid it gives a 404 error.

Reply:
```suggestion
          <li><a href="/contribute/development" class="text-light">Development</a></li>
```

Reply:
Thanks @cwhite911 ! In https://github.com/OSGeo/grass-website/pull/530/commits/5d6ff83c272d59be11a6d2b2329739d4d4d29631 I updated the links under Contribute. Since Development page is not really "developed", I put how to git, style guide and translate.